css/styleguide.scss

```diff
@@ -183,6 +183,23 @@ body {
   }
 }
 
+.site-banner-news {
+  align-items: center;
+  background-color: $color-visited;
+  color: $color-white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 10px 40px;
+
```

Comment:
I suggest stripping the y margin from the `p` at mobile and adding it back at `$medium-screen`

<img width="459" alt="screen shot 2018-11-09 at 8 59 31 am" src="https://user-images.githubusercontent.com/11464021/48276851-e56a9a80-e3fd-11e8-88b3-28193873f3c7.png">